app/components/time_ago_component.ts

```diff
@@ -0,0 +1 @@
+import {TimeAgoElement} from '@github/time-elements'
```

Comment:
for the future, shall we setup eslint for this project?

Reply:
eslint would be a fantastic idea!